package.json

```diff
@@ -67,47 +67,47 @@
     "react-dom": "^18.0.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.22.10",
+    "@babel/core": "^7.22.11",
     "@babel/plugin-proposal-class-properties": "^7.18.6",
     "@babel/plugin-proposal-object-rest-spread": "^7.20.7",
     "@babel/plugin-syntax-dynamic-import": "^7.8.3",
     "@babel/preset-env": "^7.22.10",
     "@babel/preset-react": "^7.22.5",
-    "@babel/preset-typescript": "^7.22.5",
-    "@babel/types": "^7.22.10",
+    "@babel/preset-typescript": "^7.22.11",
+    "@babel/types": "^7.22.11",
     "@commitlint/cli": "^17.7.1",
     "@commitlint/config-conventional": "^17.7.0",
     "@mdx-js/react": "^2.3.0",
-    "@storybook/addon-a11y": "^7.3.2",
-    "@storybook/addon-essentials": "^7.3.2",
-    "@storybook/addon-mdx-gfm": "^7.3.2",
-    "@storybook/addons": "^7.3.2",
-    "@storybook/blocks": "^7.3.2",
+    "@storybook/addon-a11y": "^7.4.0",
```

Comment:
The neat part about version ranges is that we do not need to update it here all the time, If you would run `npm update` it would pick the newest version matching this version range which includes 7.4.0. On an `npm install` however it would use whatever is defined in the lockfile.

`^` in the beginning means everything until the next major version in npm: https://semver.npmjs.com/#syntax-examples

Reply:
```sh
➜  ui-kit git:(main) ✗ npm ls @storybook/addon-a11y
@d2iq/ui-kit@0.0.0-dev+semantically-released /home/maex/projects/work/ui-kit
└── @storybook/addon-a11y@7.3.2

➜  ui-kit git:(main) ✗ npm update

added 448 packages, removed 305 packages, changed 749 packages, and audited 2818 packages in 2m

424 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
➜  ui-kit git:(main) ✗ npm ls @storybook/addon-a11y
@d2iq/ui-kit@0.0.0-dev+semantically-released /home/maex/projects/work/ui-kit
└── @storybook/addon-a11y@7.4.0

➜  ui-kit git:(main) ✗ git status
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   package-lock.json

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        .envrc
        devbox.json
        devbox.lock

no changes added to commit (use "git add" and/or "git commit -a")
➜  ui-kit git:(main) ✗
```

Reply:
Yep thanks! I am aware how that works. I run npm-check-updates anyway just to update all of the versions by habit. I'll keep that in mind for next time. I was going to include a major bump for a dependency here as well but it will require additional work.

Reply:
If we always update these in our `package.json` we lose the main advantage of version ranges and that is that consumers don't need to use the exact version if it can work with a slightly older one.